IRescue/Unity/Assets/Scripts/Unity/ObjectPlacing/States/SaveState.cs

```diff
@@ -4,17 +4,143 @@
 
 namespace Assets.Scripts.Unity.ObjectPlacing.States
 {
+    using System;
+    using System.Xml;
+    using UnityEngine;
+    using UnityEngine.UI;
+
     /// <summary>
     /// State when saving the game.
     /// </summary>
     public class SaveState : AbstractState
     {
         /// <summary>
+        /// The folder where the save files are located from Unity/
+        /// </summary>
+        private const string SaveFile = "Saves/";
```

Comment:
We have a const for the save files folder in 2 different files, is there a way we could just have one place where we define the save files folder?